tests/unit/deployers/gcp/test_gcp.py

```diff
@@ -1,8 +1,12 @@
 import pytest
+import re
 import fairing
 
+import httplib2
 from kubernetes import client
 from fairing.deployers.gcp.gcp import GCPJob
+from unittest.mock import patch
+from googleapiclient import discovery
```

Comment:
unused imports?

Reply:
Ack. Initially the tests were here and then I moved them out but forgot to delete the imports